test/UniswapAnchoredView.test.ts

```diff
@@ -93,10 +94,14 @@ async function setup({ isMockedView }: SetupOptions) {
           },
         ]);
         const fakeErc20 = await smock.fake([]);
-        fakeCToken.underlying.returns(() => {
-          // return another mock token as the underlying
-          return fakeErc20.address;
-        });
+        if (symbol !== "ETH") {
```

Comment:
@alexroan So I added this if check, which caused this [this](https://github.com/smartcontractkit/open-oracle/blob/feat/52055/-fetch-underlying-price-using-ctoken/test/UniswapAnchoredView.test.ts#L508) test to fail when `getUnderlyingPrice` was calling the `underlying` function..  My change [here](https://github.com/smartcontractkit/open-oracle/pull/14/files#diff-910d29ce5f6e4bd4a6c5e529ee78206138475f095f7b8093d1566e4f755f9b6cR137) got the test to pass.  Is this not sufficient to test fetching the price of `cETH`?